packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/portal/PortalControllerIntegrationTests.java

```diff
@@ -128,14 +142,87 @@ public void givenAnArbitrarySemanticElementWhenCreatingPortalOnItThenEmptyPortal
                     assertThat(portal.getLabel()).isEqualTo(SAMPLE_PORTAL);
                     assertThat(portal.getViews()).isEmpty();
                     assertThat(portal.getLayoutData()).isEmpty();
-                }, () -> fail("Missing portal"));
+                }, () -> fail(MISSING_PORTAL));
 
         StepVerifier.create(flux)
                     .consumeNextWith(initialPortalContentConsumer)
                     .thenCancel()
                     .verify(Duration.ofSeconds(10));
     }
 
+    @Test
+    @DisplayName("Given an arbitrary semantic element, when creating two portal on it, then we can move one inside the other")
```

Comment:
Try to start with as much as possible of the initial state persisted in the database. Here, you do not care about creating the two portals, just about adding one into the other.